aten/src/ATen/core/Tensor.h

```diff
@@ -27,6 +27,7 @@ namespace at {
 
 class Tensor;
 using TensorList = ArrayRef<Tensor>;
+struct Quantizer;
```

Comment:
is this necessary?  I could see it being necessary once we add a method that takes / returns a Quantizer, but we aren't there yet.

Reply:
Sure, we can add it when we expose Quantizer